### DISCUSSION TITLE
[Feature] Keep screen awake while backing up

### DISCUSSION THREAD

**Discussion by alindskog** [immich-app/immich#8217] [Feature Request]
### The feature

Hello devs!

First and foremost, let me thank you for all your hard work making such a great application! 👍 
I'm a new user and have been blown away by the level of polish in this project.

I have however noticed a few minor quality of life issues.

One thing that is making the initial upload of all my assets a bit challenging is that the iOS App doesn't prevent the device from going to sleep while on the "Backup" screen with active backup running.

Offering a "keep awake" (during active backups) setting or an "overnight" mode (that could dim the screen) is pretty common practice in similar applications and is a big help for the initial on-boarding.

Thanks for taking the time to read my suggestion!

I look forward seeing this project evolve.

Thanks,
Alex

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
We have a PR waiting to get cleaned up and merged for this feature #7777

**Reply by alindskog** [reply]
Fantastic! 🎉


(Sorry for the noise, I searched for "awake" before posting and came up empty)
